spec/order_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'order'
+
+describe Order do
+
+  let(:menu_klass) {double :menu_klass}
+  let(:menu) {double :menu}
+
+  subject(:order) {described_class.new(:menu)}
+
+  before do
+    allow(menu).to receive(:new)
+    allow(menu).to receive(:duplicate_menu).and_return :item_1 => 2, :item_2 => 2
+    subject.select_meal(:item_1)
+    subject.select_meal(:item_2)
+  end
+
+  describe '#display_menu' do
+    it 'displays a menu' do
+      expect(subject.display_menu).to eq :item_1 => 2, :item_2 => 2
+    end
+  end
+
+  describe '#select_meal' do
+    it "should reject meals not listed on the menu" do
+      expect{subject.select_meal :item_3 }.to raise_error "Please select a dish from the menu."
+    end
+    it "a valid dish should be added to my order" do
+      expect{order.my_order}.to eq :item_1 => 1, :item_2 => 1
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)